Podfile

```diff
@@ -3,9 +3,7 @@ use_frameworks!
 
 target 'AzureCalling' do
 
-pod 'AzureCommunicationCalling', '1.0.0-beta.9'
-pod 'AzureCommunication', '1.0.0-beta.9'
-pod 'AzureCore', '1.0.0-beta.9'
+pod 'AzureCommunicationCalling', '1.0.0-beta.12'
 pod 'MSAL', '1.1.13'
```

Comment:
@waynemo, I removed the AzureCore and AzureCommunication import in the Podfile since those would be automatically managed by PodSpecs. Let me know if there is any problem